src/ItemsStorage.php

```diff
@@ -214,13 +271,47 @@ public function getParents(string $name): array
         return $this->getItemsIndexedByName($rawItems);
     }
 
-    public function getChildren(string $name): array
+    public function getDirectChildren(string $name): array
+    {
+        $quoter = $this->database->getQuoter();
+        $quotedJoinColumn = $quoter->quoteTableName($this->tableName) . '.' . $quoter->quoteColumnName('name');
+        /** @psalm-var RawItem[] $rawItems */
+        $rawItems = (new Query($this->database))
+            ->select($this->tableName . '.*')
+            ->from($this->tableName)
+            ->leftJoin(
+                $this->childrenTableName,
+                [$this->childrenTableName . '.child' => new Expression($quotedJoinColumn)],
```

Comment:
```suggestion
                [$this->childrenTableName . '.child' => new Expression("{{{$this->tableName}}}.[[name]]")],
```

Reply:
We discussed it and decided that is acceptable to leave it as is.